mail_tracking/models/res_config_settings.py

```diff
@@ -8,3 +8,10 @@ class ResConfigSettings(models.TransientModel):
         related="company_id.mail_tracking_show_aliases",
         readonly=False,
     )
+    enable_mail_tracking_email_deletion_job = fields.Boolean(
```

Comment:
Not sure if this option is useful if we are using a dedicated cron task: we could simply archive/unarchive it. What do you think?

Reply:
This option could be useful if the deletion was triggered by the `api.autovacuum` feature of Odoo.

Reply:
Switched implementation to use `api.autovacuum` and no new cron jobs required